rgw/v2/tests/s3_swift/configs/test_bucket_policy_multiple_statements.yaml

```diff
@@ -0,0 +1,31 @@
+# script: test_bucket_policy_ops.py
+# bucket policy with multiple statements
+# polarion id: CEPH-11216
+config:
+  objects_count: 25
+  objects_size_range:
+    min: 5
+    max: 15
+  test_ops:
+    upload_type: normal
+    verify_policy: True
+    policy_document:
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Action": ["s3:GetBucketPolicy"],
```

Comment:
@hmaheswa , can we test more s3 operations, like PutBucketPolicy, DeleteBucketPolicy,CreateMultipartObjects, DeleteObjects, PutBucketVersioning, and Getbucketversioning? 

You may have one config, but can define separate modules for all the above scenarios in a separate python file  at location reusables/test_s3_operations.py ?

Reply:
This PR Looks good for the test case CEPH-11217 that addresses the conflicting policy statements.

Reply:
sure @viduship 
working on it

Reply:
@viduship removed multiple statements config from this PR, will create a seperate PR for it
This PR is only for conflicting statements